### DISCUSSION TITLE
[ENHANCEMENT] Have the option to set the scanning speed of the artifact same as scan material

### DISCUSSION THREAD

**Discussion by husky0723** [Andrewthe13th/Inventory_Kamera#375] [Ideas or Feature requests]
Maybe is just me but I notice sometime when you or the scanner select things too fast in game, the game will ignore the input thus cause scanner skip to record the artifact. Even though we have the 100ms option but sometime the game still fail to register the input. However material scanning doesn't face this problem due to slower select speed. Which is why I propose to add an option to have the same scanning speed as the material scanning

**Comment by Cupcak3** [comment]
Thanks for your input. I'll look into adding options for longer delays.

**Comment by Cupcak3** [comment]
The 'slow' delay in the newest release should be long enough for most computers that are not as powerful as other computers. I have a pretty strong one myself.
